.github/CODE_STYLE.md

```diff
@@ -118,6 +118,13 @@ export class TdsComponent {
 
 ```
 
+
+## Slots
+
+Slots are used throughout many of our components. Some of these are named slots, and in that case they should follow
+to following convention: The slot should always be named to represent it position/use case. For example this could be
```

Comment:
```suggestion
the following convention: The slot should always be named to represent its position/use case. For example this could be
```

Reply:
I can see that Tim made this same comment and that you merged it, but it doesn't reflect in GitHub or when I checkout the branch locally. Please feel free to resolve this if it has been fixed 👍